rules/windows/process_creation/proc_creation_win_susp_reg_add.yml

```diff
@@ -0,0 +1,24 @@
+title: Reg Add Suspicious Path 
+id: b7e2a8d4-74bb-4b78-adc9-3f92af2d4829
+status: experimental
+description: An adversary may use many registry path with reg.exe 
```

Comment:
Detects when an adversary uses the 'AppDataLow' subkeys as a place to store data as seen in the URSNIF phishing campaign

Reply:
The idea was to add more generic suspicious path. 
Will change If there are more added.

Reply:
Ah, sorry then didn't understand that. Good idea 👍

Reply:
Let's add suspicious folders right waway

Reply:
I'll try to find more with this query : https://www.google.de/search?q=inurl%3Aany.run+%22reg+add%22+%5CSoftware%5CAppDataLow%5C

Reply:
Okay, it's not that easy. We'll leave it as it is for now.